core/controllers/collection_viewer.py

```diff
@@ -39,6 +46,19 @@ class CollectionDataHandler(base.BaseHandler):
     """Provides the data for a single collection."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'collection_id': {
+            'type': 'basestring'
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'v': {
+                'type': 'basestring',
+                'default_value': None
+            }
```

Comment:
Why is this schema needed when the arg 'v' is not being used?

Reply:
Yes, in the backend test of collection_editor it is referenced. 
Here: https://github.com/oppia/oppia/blob/73936fd34536b51d1e29e4b4e8740741706ef5dd/core/controllers/collection_editor_test.py#L81

Reply:
I think there might be some inconsistency in the code. @aks681 what do you think about this?